### PR TITLE
dotnet/templating#3360 using file-scoped namespaces in classlib template

### PR DESCRIPTION
**Pull request by vlada-shubina**
### Problem
#3360 using file-scoped namespaces

### Solution
Added using of file-scoped namespaces in classlib template

### CODE REVIEW

File: template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ClassLibrary-CSharp/Class1.cs [hunk 1]
```diff
@@ -1,8 +1,7 @@
-﻿using System;
+﻿namespace Company.ClassLibrary1
+using System;
```
Comment (DamianEdwards) [inline template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ClassLibrary-CSharp/Class1.cs:2]: I think we should keep the ordering as it was, i.e.:
``` csharp
using System;

namespace Company.ClassLibrary1;

public class Class1
{
}
```

Comment (vlada-shubina) [reply template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ClassLibrary-CSharp/Class1.cs:2]: is correct syntax `namespace Company.ClassLibrary1;` or `namespace Company.ClassLibrary1` (note semicolon)?

Comment (DamianEdwards) [reply template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ClassLibrary-CSharp/Class1.cs:2]: My understanding is the semicolon is required: https://github.com/dotnet/csharplang/blob/main/proposals/csharp-10.0/file-scoped-namespaces.md

**Comment by DamianEdwards** [comment]
Note [this comment](https://github.com/dotnet/aspnetcore/pull/34220#issuecomment-879479539) RE the compiler feature to enable file-scoped namespaces. In short, it wasn't flowing properly but is now.

**Comment by DamianEdwards** [comment]
The compiler with support for this is now in SDK version 6.0.100-preview.7.21364.4

**Review by DamianEdwards — APPROVED** [review]

**Review by DavidKarlas — APPROVED** [review]

**Pull request merged**